cmd/cli/commands/deals.go

```diff
@@ -234,40 +235,78 @@ var dealQuickBuyCmd = &cobra.Command{
 	},
 }
 
+func getBlacklistType() (pb.BlacklistType, error) {
```

Comment:
Rewrite using the following algorithm:
1. Uppercase.
2. Prepend `BLACKLIST_`.
3. Use either `proto.UnmarshalJSONEnum` or `BlacklistType_value`.

Then it will be more flexible to future changes.

Reply:
you know, it would be out of the scope, because code was just extracted unchanged, but ok, i'll do it